src/App.js

```diff
@@ -0,0 +1,23 @@
+import { HashRouter, Switch, NavLink, Route, Redirect } from "react-router-dom";
+import MovieList from "./core/MovieList";
+import People from "./features/personList";
+import { toMovieListPage, toPeople } from "./routes";
+import { Header } from "./common/Header";
+
+export default () => (
+    <HashRouter>
+        <Header />
```

Comment:
Would be so good idea to change that 'Header' to 'Navigation' or simply 'Nav'
```suggestion
        <Navigation />
```